AdobeStockClient/Model/ConnectionFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\AdobeStockClient\Model;
+
+use AdobeStock\Api\Client\AdobeStock;
+
+/**
+ * Class ConnectionFactory
+ */
+class ConnectionFactory
+{
+    /**
+     * @param string $apiKey
+     * @param string $productName
+     * @param string $targetEnvironment
+     *
+     * @return AdobeStock
+     */
+    public function createConnection(string $apiKey, string $productName, string $targetEnvironment): AdobeStock
```

Comment:
Let's rename to `create`, we already have `ConnectionFactory` in the class name, it would be better to simplify the method name.

Reply:
Fixed